__tests__/api.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../api/app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const categories = require("../db/data/test-data/categories");
+
+afterAll(() => {
+  return db.end();
+});
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("/api", () => {
+  test("GET - Status: 200 - returns an object with message 'all ok'", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toEqual({ message: "all ok" });
+      });
+  });
+});
+
+describe("/api/categories", () => {
+  test('GET - Status: 200 - returns an array of category objects, each of which having the properties "slug", "description"', () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        response.body.categories.forEach((category) => {
+          console.log(category);
+          expect(typeof category.description).toBe("string");
```

Comment:
It's good you're checking that this will be a string, but can you be more specific? What exactly are you expecting the string to be?